scripts/bootstrap.ps1

```diff
@@ -342,6 +344,10 @@ function Install-Rust {
   Add-To-Path "$rustPath\cargo\bin"
 }
 
+function Install-PdbAddr2line {
+  Execute-Command cargo install --examples "pdb-addr2line@0.11.2"
```

Comment:
is `--examples` necessary?

Reply:
they should be using https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries but good to know